src/sentry/static/sentry/app/components/panels/panelHeader.jsx

```diff
@@ -2,9 +2,10 @@ import {Flex} from 'grid-emotion';
 import PropTypes from 'prop-types';
 import React from 'react';
 import styled, {css} from 'react-emotion';
+import space from '../../styles/spacingScale';
```

Comment:
Can we make the file name == default import name?

Reply:
yep. I would prefer calling is just `space` so that lines like `margin: space(3) space(0.5)` don't get super long. I think since `space.jsx` is in the `styles` directory it should all be fairly clear.

Reply:
we could do `scale` but we would need to make sure it also works for type, otherwise it's ambiguous. 

with type we've gotta have low-increment values, especially around the base unit (12px, 18px, 24px) etc so it might be best to do a second scale for that.

also who doesn't love talking about space?

![image](https://user-images.githubusercontent.com/435981/38326126-d2908ac4-37f9-11e8-9e4a-636775653278.png)